### ISSUE TITLE
HUBS and ORG Users should not have access to create Surveys from the app

### ISSUE THREAD

**Issue by stacylorraineprendeville** [FundacionParaguaya/MentorApp#866]
As a HUB or ORG user
I should not be able to access the app to create a survey
ONLY mentor roles can create surveys on the platform
THIS needs to be reflected in the APP
The app must function as the platform does.
Therefore only mentor users can access the APP.

**Comment by Danman7** [comment]
This will happen based on `authorities.authority` prop we get when user logs in. Slight modification of what we store in redux is needed. User will access lifemap creation only if `ROLE_SURVEY_USER`.

**Comment by stacylorraineprendeville** [comment]
@loorely might be relevant for your upcoming user role

**Comment by aprojas** [comment]
Maybe this might be useful to know:

Logged in with my *Fundacion Paraguaya hub* in the mentor app
1. I could finish a survey, but it does not sync. Just shows the red label "sync"
2. BUT the family does appear in the families list
3. The dashboard shows 0 families, but does show a number of greens, yellows and reds.. 

![Screenshot_2019-07-04-08-33-17.png](https://images.zenhubusercontent.com/5c48c8dbf883486cde30d849/fa7c2843-a463-4f9e-ad39-636479626406)

![Screenshot_2019-07-04-08-33-12.png](https://images.zenhubusercontent.com/5c48c8dbf883486cde30d849/0200fd07-76f4-4ae8-bb30-a4a1a903a890)

![Screenshot_2019-07-04-08-33-36.png](https://images.zenhubusercontent.com/5c48c8dbf883486cde30d849/f92d2e01-3cb9-41fb-a19e-031126c2c5fe)

**Comment by Danman7** [comment]
@stacylorraineprendeville @aprojas We have no idea of what hubs and orgs are. We just send a login request to the server and it reponds with OK you are logged in. Can we discuss this togethere with some backend so we can implement it?

**Comment by stacylorraineprendeville** [comment]
@danman7 I just do not understand how this gets past grooming and recieves mulitple comments on approach and then gets returned like this.
Please do speak to the back end

**Comment by Danman7** [comment]
This is ready to be implemented.

**Comment by diegotorrespy** [comment]
@Danman7 
This is part of the response from the back end for the access token request for a mentor user:

```json
{
  "access_token": "51074b53-965d-4c11-9f73-c7a180e1b778",
  "token_type": "bearer",
  ...
    "authorities": [
      {
        "authority": "ROLE_SURVEY_USER"
      }
    ],
    "accountNonLocked": true,
    "credentialsNonExpired": true,
    "accountNonExpired": true
  }
}
```
And the `authorities` object can take the next values:
- ROLE_ROOT: for admin accounts.
- ROLE_PS_TEAM: for poverty stoplight teams accounts.
- ROLE_HUB_ADMIN: for hubs accounts.
- ROLE_APP_ADMIN: for orgs accounts
- ROLE_SURVEY_USER: for mentors

**Comment by pA1nD** [comment]
After talking to Stacy, it's clear now that the user should not at all be able to login, unless it's a Mentor.

Behavior would be to log the user out and show an error message:
"Only Mentors can access the app"

I think we should show this error also in spanish?

@aprojas could you maybe provide a translation?

**Comment by aprojas** [comment]
"Only Mentors can access the app"
"Únicamente los Encuestadores pueden acceder a la aplicación"

@pA1nD

**Comment by stacylorraineprendeville** [comment]
@pA1nD @aprojas The final decision from Kathi is to use the word "facilitator" as not all partners use mentors - please amend this or ingest it
As for this error message I would not give this message - 
First - Will we know if it is an ORG or HUB or can this just apply to anyone who does not have access rights (i.e. false accounts) let me know

**Comment by pA1nD** [comment]
@aprojas could we have adjusted tranlsation in this case?

**Comment by aprojas** [comment]
@pA1nD but Stacy just commented that this message should not be shown. @stacylorraineprendeville

**Comment by pA1nD** [comment]
Sorry. @aprojas @stacylorraineprendeville , We would only show the error message, if the user who tries to login is not a mentor.

**Comment by aprojas** [comment]
If the above text will be shown, then it should be:

"Only facilitators can access the app"
"Únicamente los facilitadores pueden acceder a la aplicación"

@pA1nD

**Comment by pA1nD** [comment]
@zommerberg here you have the translations. ✌️

**Comment by aprojas** [comment]
/If the above text will be shown, then it should be:

"Only facilitators can access the app"
"Solo los facilitadores pueden acceder a la aplicación"/


Are we going to change these translations for the next release?
@zommerberg @pA1nD @Danman7

**Comment by zommerberg** [comment]
this is already merged with develop in #1030
